milkyway/chain.json

```diff
@@ -0,0 +1,154 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "milkyway",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://milkyway.zone/",
+  "pretty_name": "MilkyWay",
+  "chain_type": "cosmos",
+  "chain_id": "milkyway",
+  "bech32_prefix": "milk",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md